config/test.exs

```diff
@@ -25,8 +25,15 @@ config :alloy_ci, AlloyCi.Notifier,
   email: "false"
 
 # Configure your database
+# username: "postgres",
+  # password: "postgres",
+  # database: "alloy_ci_dev",
+  # hostname: "localhost",
+  # pool_size: 10
 config :alloy_ci, AlloyCi.Repo,
   adapter: Ecto.Adapters.Postgres,
-  url: System.get_env("DATABASE_URL"),
+  # url: System.get_env("DATABASE_URL"),
```

Comment:
Please keep this one as an env variable, because it messes with the CI configurations otherwise.